velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -1296,49 +739,48 @@ void JoinFuzzer::verify(core::JoinType joinType) {
     }
   }
 
-  std::vector<PlanWithSplits> altPlans;
-  altPlans.push_back(makeDefaultPlan(
-      joinType,
-      nullAware,
-      probeKeys,
-      buildKeys,
-      flatProbeInput,
-      flatBuildInput,
-      probeProjections,
-      buildProjections,
-      outputColumns,
-      filter));
+  std::vector<JoinMaker::PlanWithSplits> altPlans;
 
-  makeAlternativePlans(
-      defaultPlan.plan,
-      convertedProbeInput,
-      convertedBuildInput,
-      probeProjections,
-      buildProjections,
-      altPlans,
-      filter);
-  makeAlternativePlans(
-      defaultPlan.plan,
-      flatProbeInput,
-      flatBuildInput,
-      probeProjections,
-      buildProjections,
-      altPlans,
-      filter);
+  addPlansForInputType(JoinMaker::InputType::ENCODED, joinMaker, altPlans);
+  addPlansForInputType(JoinMaker::InputType::FLAT, joinMaker, altPlans);
 
-  addPlansWithTableScan(
-      tableScanDir->getPath(),
-      joinType,
-      nullAware,
-      probeKeys,
-      buildKeys,
-      flatProbeInput,
-      flatBuildInput,
-      probeProjections,
-      buildProjections,
-      outputColumns,
-      altPlans,
-      filter);
+  if (joinMaker.supportsTableScan()) {
+    const int32_t numGroups = randInt(1, probeSource->batches().size());
+
+    altPlans.push_back(joinMaker.makeHashJoinWithTableScan(
+        std::nullopt, // Use ungrouped execution.
```

Comment:
nit: Move the comment to a separate line above the code? Ditto for the comment below.